### PR TITLE
Switch to IAuth

### PR DESCRIPTION
**Pull request by Mbrick2**
[Here are the docs](https://auth.itinerary.eu.org/documentation/migrating-from-fluffyscratch). It looks a lot nicer and easier. Also less downtime

### CODE REVIEW

File: src/routes/auth/verify.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { createSession, getUserByUsername, createUser } from '$lib/db';
 export async function post({ request }) {
   const { privateCode } = await request.json();
 
-  const res = await fetch('https://fluffyscratch.hampton.pw/auth/verify/v2/' + privateCode);
+  const res = await fetch('https://auth.itinerary.eu.org/api/auth/verifyToken?privateCode=' + privateCode = "&name=Aviate");
```
Comment (Looky1173) [inline src/routes/auth/verify.js:8]: You have to add `&name=Aviate` to the other API endpoint.

File: src/routes/login.svelte [hunk 2]
```diff
@@ -26,7 +26,7 @@
     const privateCode = urlParams.get('privateCode');
 
     const base64 = btoa(location.host + '/login');
-    link = `https://fluffyscratch.hampton.pw/auth/getKeys/v2?redirect=${base64}`;
+    link = `https://auth.itinerary.eu.org/auth/?redirect=${base64}`;
```
Comment (Looky1173) [inline src/routes/login.svelte:29]: The `name` parameter goes here: `https://auth.itinerary.eu.org/auth/?redirect=${base64}&name=Aviate`

File: src/routes/auth/verify.js (hunk 1 above)
Comment (MystPi) [reply src/routes/auth/verify.js:8]: I'll fix it.

**Comment by MystPi** [comment]
Awesome! It's working great; I'll merge.

**Comment by Looky1173** [comment]
@MystPi Thanks for being an early adopter of Scratch Auth! Also, thanks @Mbrick2 for the pull request.

**Pull request merged**